src/main.rs

```diff
@@ -252,6 +254,10 @@ fn run_cli<W: Write, R: Read, L: OutputPin, SPI, PADS, const I: usize>(
                     Base::Boot => {
                         run_payload();
                     }
+                    Base::Print => {
+                        // Print the information about the configs.bootargs variable
+                        writeln!(d.tx, "configs.bootargs = {:?}", _c.bootargs).ok();
```

Comment:
The `bootarg` argument should be dumped as a UTF-8 string other than a slice. Please convert `c.bootargs` into a UTF-8 string and avoid using debug formatter `{:?}` here. The variable `_c` should be renamed to `c`, as the underscore prefix `_` is not necessary if a variable is used. The underscore prefix `_` should only be used if we want to suppress a warning on unused variables.

Reply:
Thanks for the correction, I will fix it latter